process-update.js

```diff
@@ -65,12 +40,44 @@ module.exports = function(hash, moduleMap, options) {
         return null;
       }
 
+      var failedUpdate = false;
```

Comment:
failedUpdate is false here then how it will get changed to true again when the update gets failed.

Reply:
line 62, 68, 79 in the new coe.

Reply:
Understood Thank you